libs/estdlib/src/gen_statem.erl

```diff
@@ -222,16 +223,8 @@ handle_cast(Request, State) ->
     do_handle_state(cast, Request, State).
 
 %% @hidden
-handle_info(
-    {timeout, _TimerRef, {state_timeout, State, Msg}},
-    #state{current_state = CurrentState} = State
-) ->
-    case State of
-        CurrentState ->
-            do_handle_state(state_timeout, Msg, State);
-        _ ->
-            ok
-    end;
+handle_info({timeout, _TimerRef, {state_timeout, CurrentState, Msg}}, #state{current_state=CurrentState} = State) ->
```

Comment:
Do you mind running `rebar3 fmt -w src/gen_statem.erl` so changes are already formatted?